app/models/invoice_item.rb

```diff
@@ -7,4 +7,8 @@ class InvoiceItem < ApplicationRecord
   def formatted_invoice_date
     invoice.created_at.strftime("%A, %B %d, %Y")
   end
+
+  def formatted_unit_price_invoice_item
+    "$#{unit_price.to_f / 100}"
+  end
```

Comment:
We can use `number_with_precision` to format this to a string with 2 decimal places, e.g. $10.00